py/picca/delta_extraction/masks/absorber_mask.py

```diff
@@ -11,10 +11,10 @@
 
 defaults = {
     "absorber mask width": 2.5,
+    "los_id name": "THING_ID",
```

Comment:
this is mainly to allow additional flexibility later when having other surveys/cats? Instead of guessing through all known values for this? Where was this set before if we didn't allow it as an option and didn't have a default value?

Reply:
This was not working before. I called it a minor fix as it is somehow still not working (though this step is required to make this class work). When I run delta_extraction using this mask there seems to be no effect. I added an issue to track it, but I plan to address it after the holidays